src/cartridges/int_adyen_SFRA/cartridge/client/default/js/constants.js

```diff
@@ -4,4 +4,9 @@ module.exports = {
   METHOD_ADYEN: 'Adyen',
   METHOD_ADYEN_POS: 'AdyenPOS',
   METHOD_ADYEN_COMPONENT: 'AdyenComponent',
+  GIFTCARD_CONSTANTS: {
+    GIFTCARDAMOUNT: 'Giftcard Amount',
+    REMAININGAMOUNT: 'Remaining Amount',
+    CANCELGIFTCARD: 'Cancel Giftcard?',
```

Comment:
Should we use Resource files instead to also enable translations on the storefront?

Reply:
good point